### ISSUE TITLE
android studio can't find java

### ISSUE THREAD

**Issue by lapisdecor** [snapcrafters/android-studio#78]
After update to the latest flutter and dart plugins, android studio can't find java.

```Launching lib/main.dart on Android SDK built for x86 in debug mode...
Running Gradle task 'assembleDebug'...

ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.

Please set the JAVA_HOME variable in your environment to match the
location of your Java installation.

Exception: Gradle task assembleDebug failed with exit code 1```

**Comment by lapisdecor** [comment]
Also, I have a new Android folder on ~/ that I thinks wasn't previously there.

**Comment by lapisdecor** [comment]
I had it working on another machine before the android studio and flutter updates. I did an update on that machine too and run into problems. The problem may not be the snap but the explosive combination of java / gradle versions and android studio not reseting (it assumes old configuration). I got android studio working after erasing .config/Google and .local/share/Google but I still run into problems with gradle < 6 (which I think is a flutter problem). Java version must be openjdk not oracle and <= 11 for things to work.
